src/etc/inc/plugins.inc.d/webgui.inc

```diff
@@ -411,14 +411,29 @@ ssl.openssl.ssl-conf-cmd = (
 
 EOD;
         } else {
-            $lighty_config .= <<<EOD
-
-ssl.openssl.ssl-conf-cmd = (
-    "MinProtocol" => "TLSv1",
-    "CipherString" => "{$config['system']['webgui']['ssl-ciphers']}"
-)
-
-EOD;
+            $tls13_suites = array("TLS_AES_128_GCM_SHA256", "TLS_AES_256_GCM_SHA384", "TLS_CHACHA20_POLY1305_SHA256", "TLS_AES_128_CCM_SHA256", "TLS_AES_128_CCM_8_SHA256");
```

Comment:
Personally hardcoding RFCs here is what we wanted to avoid with these advanced selectors. I’m not a fan of the setting as a whole, but it also seems we can’t do much else here other than fix it like this.

Reply:
agree. and i thought about ` openssl ciphers -s -tls1_3 `. but it returns only 3 of 5 suits.
although with a direct request it gives all 5
`openssl ciphers -tls1_3 -v -s -ciphersuites TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:TLS_AES_128_CCM_8_SHA256:TLS_AES_128_CCM_SHA256`

Reply:
That’s slightly better, hopefully LibreSSL doesn’t screw this up. Still have to deal with this as well until 23.1.

Reply:
actualy, system_advanced_admin.php doesn't show these two suites (`TLS_AES_128_CCM...`) either. so we realy can try to use `configd_run("system ssl ciphers")` (like system_advanced_admin.php do)  instead of static array (will this call work on system start)?